makerspace/templates/tools/admin_tool.html

```diff
@@ -0,0 +1,90 @@
+{% extends "web/base.html" %}
+{% load staticfiles %}
+{% load i18n %}
+{% load news_tags %}
+
+{% block head %}
+    <link rel="stylesheet" type="text/css" href="{% static "news/css/admin.css" %}">
+{% endblock head %}
+
+{% block body %}
+    <h1 class="ui text-centered make_col_yellow header">
+        {% trans "Tools" %}
+        <a class="make_col_blue explanation-popup" data-title="{% trans "Create" %}"
+           data-content="{% trans "If clicked, a new Tool will be created." %}" href="{% url 'makerspace-tools-create' %}">
```

Comment:
Maybe replace the tooltip with something like "Create a new tool"? It is a button, the clicking part is implicit.